minichain/rpc.py

```
@@ -0,0 +1,87 @@
+                tx = Transaction.from_dict(tx_data)
+                if not tx.verify():
+                    raise ValueError("Invalid signature")
+                if self.mempool.add_transaction(tx):
+                    asyncio.create_task(self.network.broadcast_transaction(tx))
+                    result = tx.tx_id
+                else:
+                    raise ValueError("Transaction rejected by Mempool")
+            else:
+                return {"jsonrpc": "2.0", "error": {"code": -32601, "message": f"Method not found: {method}"}, "id": req_id}
+
+            return {"jsonrpc": "2.0", "result": result, "id": req_id}
```

Ruff (0.15.17)

[warning] Consider moving this statement to an `else` block

(TRY300)